src/Integration.UnitTests/Helpers/SonarLintOutputTests.cs

```diff
@@ -41,58 +42,117 @@ public void Write_OutputsToWindow()
         {
             // Arrange
             var windowMock = new ConfigurableVsOutputWindow();
+            var sonarLintSettings = CreateSonarLintSettings(DaemonLogLevel.Info);
+            var serviceProviderMock = CreateConfiguredServiceProvider(windowMock);
 
-            var serviceProviderMock = new ConfigurableServiceProvider(assertOnUnexpectedServiceRequest: true);
-            serviceProviderMock.RegisterService(typeof(SVsOutputWindow), windowMock);
-
-            var logger = new SonarLintOutputLogger(serviceProviderMock, Mock.Of<ISonarLintSettings>());
+            var testSubject = CreateTestSubject(serviceProviderMock, sonarLintSettings);
 
             // Act
-            logger.WriteLine("123");
-            logger.WriteLine("abc");
+            testSubject.WriteLine("123");
+            testSubject.WriteLine("abc");
 
             // Assert
             var outputPane = windowMock.AssertPaneExists(VsShellUtils.SonarLintOutputPaneGuid);
             outputPane.AssertOutputStrings("123", "abc");
         }
 
         [TestMethod]
-        [DataRow(DaemonLogLevel.Info, false)]
-        [DataRow(DaemonLogLevel.Minimal, false)]
-        [DataRow(DaemonLogLevel.Verbose, true)]
-        public void LogVerbose_OutputsToWindowIfLogLevelIsVerbose(DaemonLogLevel logLevel, bool shouldLogMessage)
+        [DataRow(DaemonLogLevel.Info)]
+        [DataRow(DaemonLogLevel.Minimal)]
+        [DataRow(DaemonLogLevel.Verbose)]
+        public void LogVerbose_OnlyOutputsToWindowIfLogLevelIsVerbose(DaemonLogLevel logLevel)
```

Comment:
Renamed and simplified the existing test.
The test name says it's testing `LogVerbose` and only expects output if the level is `Verbose`, so there's no need for the `shouldLogMessage` parameter.